cdap-integration-test/src/test/java/co/cask/cdap/test/IntegrationTestBaseTest.java

```diff
@@ -29,6 +29,9 @@
  */
 public class IntegrationTestBaseTest extends IntegrationTestBase {
 
+  public static final String TEST_APPLICATION_NAME = "TestApplication";
+  public static final String FLOWLET_NAME = "theFlowlet";
```

Comment:
The string "theFlowlet" is also used here:
https://github.com/caskdata/cdap/blob/feature/changes-to-integrationtestbase/cdap-integration-test/src/test/java/co/cask/cdap/test/TestFlow.java#L33

I'd suggest moving the constant to that class, so you also use the constant there.